config.toml.example

```diff
@@ -313,8 +313,9 @@
 # compiler.
 #codegen-units = 1
 
-# Sets the number of codegen units to build the standard library with,
-# regardless of what the codegen-unit setting for the rest of the compiler is.
+# Sets the number of codegen units to build the standard library (including
+# core, liballoc, compiler_builtins etc.) with, regardless of what the
```

Comment:
I'm not happy with the "etc." part here -- it'd be great if we could be more specific and say *exactly* where this applies, but I don't know the answer myself..

Reply:
Everything in `library/` nowadays I think

Reply:
I think this setting is for some of the libs, not all. For example if I build with `codegen-units-std = 1` but without `codegen-units = 1` I get multiple object files for `compiler_builtins`.

Reply:
compiler-builtins is special cased, it's always intended to compile into basically every-function-is-a-codegen-unit I believe, for linkage reasons.